src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -491,7 +491,17 @@ private List<LaunchCommand> GetInitializeCommands()
             string escappedSearchPath = string.Join(pathEntrySeperator, _launchOptions.GetSOLibSearchPath().Select(path => EscapePath(path, ignoreSpaces: true)));
             if (!string.IsNullOrWhiteSpace(escappedSearchPath))
             {
-                commands.Add(new LaunchCommand("-gdb-set solib-search-path \"" + escappedSearchPath + pathEntrySeperator + "\"", ResourceStrings.SettingSymbolSearchPath));
+                if (_launchOptions.DebuggerMIMode == MIMode.Gdb)
+                {
```

Comment:
@chuckries @gregg-miskelly @caslan Please re-review this line. clrdbg and lldb don't match gdb here.